dex/testing/eth/harness.sh

```diff
@@ -182,7 +188,21 @@ do
     break
   fi
   echo "Waiting for light nodes to sync."
-  sleep 5
+  # Although not necessary here, mine while waiting so that transactions are
+  # mined if not mined yet.
+  "${NODES_ROOT}/harness-ctl/mine-alpha" "5"
+done
+
+# Transactions can take an eternity to be mined...
+# TODO: Determine why this is.
```

Comment:
Ugh. Looks like no simple solution. https://ethereum.stackexchange.com/questions/2539/how-do-i-decrease-the-difficulty-on-a-private-testnet

Would it be better to prevent the difficulty increase by delaying mining by 15 seconds between each block?

Reply:
I'm not sure this is the same problem. Mining is fine because we use oracles to create blocks, there isn't any actual mining. We are using [clique](https://eips.ethereum.org/EIPS/eip-225) in the test harness. It could be related though. Transactions just seem to get put into blocks randomly.

Reply:
Trying that, it's really more noticeable in the test in #1019